Source/Scene/GltfLoader.js

```diff
@@ -326,8 +326,8 @@ function loadVertexBuffer(
     baseResource: loader._baseResource,
     bufferViewId: bufferViewId,
     draco: draco,
-    dracoAttributeSemantic: semantic,
-    dracoAccessorId: accessorId,
+    attributeSemantic: semantic,
+    accessorId: accessorId,
```

Comment:
I renamed `dracoAttributeSemantic` and `dracoAccessorId` to more general terms `attributeSemantic` and `accessorId` since it was being used in contexts other than Draco now.